pkg/pod/pod.go

```diff
@@ -952,6 +952,33 @@ func (builder *Builder) GetLog(logStartTime time.Duration, containerName string)
 	return buf.String(), nil
 }
 
+// GetFullLog connects to a pod and fetches the full log since pod creation.
+func (builder *Builder) GetFullLog(containerName string) (string, error) {
+	if valid, err := builder.validate(); !valid {
+		return "", err
+	}
+
+	req := builder.apiClient.Pods(builder.Definition.Namespace).GetLogs(builder.Definition.Name, &v1.PodLogOptions{})
+	log, err := req.Stream(context.Background())
+
+	if err != nil {
+		return "", err
+	}
+
+	defer func() {
+		_ = log.Close()
+	}()
+
+	buf := new(bytes.Buffer)
```

Comment:
more meaning var name would help future readers.

Reply:
@trewest @achuzhoy Honestly I just copied the GetLog function and dropped the 'SinceSeconds' field. I didn't touch any variable names from what was already on pod pkg
I can of course change the variable names but maybe we should do it as well for the first function?

Reply:
IMHO The other function can (or even should) be a separate PR.